docs/sources/articles/dockerfile_best-practices.md

```diff
@@ -335,12 +331,31 @@ fi
 exec "$@"
 ```
 
-That script then gets copied into the container and run via `ENTRYPOINT` on
-container startup:
+> **Note**: This script uses [the `exec` Bash command](http://wiki.bash-hackers.org/commands/builtin/exec)
+> so that the final running application becomes the container's `PID 1` so that
+> it will receive any Unix signals sent to the container. See the
+> [`ENTRYPOINT`](https://docs.docker.com/reference/builder/#ENTRYPOINT) help for more
+> details.
+
+
+That script is copied into the container and run via `ENTRYPOINT` on
```

Comment:
"That script" possibly confusing again (I "get" the difficulty here :))

"... and run via"; should that be "... and *executed* via (/by?)" ? Not sure, may be just preference.